lib/kennel/output_limiter.rb

```diff
@@ -1,25 +0,0 @@
-# frozen_string_literal: true
-
-module Kennel
-  class OutputLimiter
-    def initialize(out, max_lines, &on_overflow)
-      @out = out
-      @max_lines = max_lines
-      @on_overflow = on_overflow
-      @seen = 0
-    end
-
-    def puts(*content)
-      content.join("\n").split("\n").each do |line|
```

Comment:
> use split(/\n/) to not remove "\n\n" cases

I'm not sure that the old code used to do that, did it?

Reply:
it did not
I did not see it causing an actual issue, just wanted the code to be technically correct

Reply:
OK, just an odd thing to include in the PR description then.

Reply:
wanted to call out the diff